src/FlightCache.test.ts

```diff
@@ -0,0 +1,32 @@
+import 'jest';
+import Generator from './Generator';
+import airports from './Data/airports';
+import FlightCache from './FlightCache';
+import { JestEnvironment } from '@jest/environment';
+
+describe('FlightCache ', () => {
+  const sampleDate = '2020-01=01';
+  const origin = airports[0];
+  const destination = airports[1];
+
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  it('should return null for a cache miss', () => {
+    const flights = FlightCache.getFlights(sampleDate);
+    expect(flights).toEqual(null);
+  });
+
+  it('should return flights for a cache hit on date', () => {
+    const numFlights = 10;
+    const generator = new Generator(sampleDate);
+    const flights = Array(numFlights).fill(null).map(() => {
+      return generator.flight(origin, destination);
+    });
+    FlightCache.cacheFlights(sampleDate, flights);
```

Comment:
Because FlightCache is a singleton, these tests pass based on the order they are run. Might be a good idea to have the ability to import the class not instantiated so you can have a different cache per test. (Or a way to reset the cache)

Reply:
That should be accomplished by what I have on line 13 above ☝️

Reply:
Ahhhh, nice nice. Totally missed that 👍